src/modules/job-manager/event.c

```diff
@@ -428,9 +489,8 @@ void event_ctx_set_start_ctx (struct event_ctx *ctx,
     ctx->start_ctx = start_ctx;
 }
 
-/* N.B. any in-flight batches are destroyed here.
- * If they are not yet fulfilled, user callbacks may synchronously block on
- * flux_future_get().
+/* Any in-flight KVS batche commits and their follow-on job-state.STATE event
```

Comment:
typo "batche"

Reply:
That one was already fixed.  Comment on an outdated branch?

Reply:
Oh sorry that was my bad - I must have squashed that into the wrong commit!